spec.md

```diff
@@ -455,22 +456,28 @@ If the query parameter described above is present, and the broker executes the r
 
 An asynchronous response triggers the platform marketplace to poll the endpoint `GET /v2/service_instances/:instance_id/last_operation` until the broker indicates that the requested operation has succeeded or failed. Brokers MAY include a status message with each response for the `last_operation` endpoint that provides visibility to end users as to the progress of the operation.
 
-#### Blocking Operations
+## Blocking Operations
 
-Brokers do not have to support concurrent requests that act on the same
-set of resources.  If a broker receives a request that it is not able to
+Brokers do not have to support concurrent requests that mutate the same
+resource.  If a broker receives a request that it is not able to
 process due to other activity being done on that resource then the broker
-MUST reject the request with an HTTP `422 Unprocessable Entity`.
-The HTTP body of the response MUST include a `description` property
-explaining the reason for the failure.
+MUST reject the request with an HTTP `409 Conflict` and
+the following body:
 
-Sample response:
 ```
 {
+  "error": "ConcurrencyError",
   "description": "Another operation for this service instance is in progress"
 }
 ```
 
+Note that per the [Orphans](#orphans) section, this error response does not
+cause orphan mitigation to be initiated. Therefore, platforms receiving
+this error response SHOULD resend the request at a later time.
+
+Brokers MAY choose to treat the creation of a binding as a mutation of
+the corresponding service instance - it is an implementation choice.
```

Comment:
Add text about how this might cause platforms to serialize binding create() requests.

Reply:
done